### PR TITLE
add arbitrary change

### PR DESCRIPTION
**Pull request by pxr687**

### CODE REVIEW

File: dunning_kruger.Rmd [hunk 1]
```diff
@@ -103,3 +103,7 @@ df
 ```{python}
 df.groupby('actual_quantile', observed=True)['difference'].mean()
 ```
+
+```{python}
+# an arbitrary change
```
Comment (matthew-brett) [inline dunning_kruger.Rmd:108]: This change seems a bit - well - arbitrary - can you make a more substantial change?

**Review by matthew-brett — COMMENTED** [review]
Thoughts?

File: dunning_kruger.Rmd (hunk 1 above)
Comment (pxr687) [reply dunning_kruger.Rmd:108]: OK I will have a think about other changes.

File: dunning_kruger.Rmd [hunk 2]
```diff
@@ -103,3 +103,21 @@ df
 ```{python}
 df.groupby('actual_quantile', observed=True)['difference'].mean()
 ```
+
+```{python}
+# add some randomness to perceived percentiles
+df['perceived_percentile_w_randomness'] = df['perceived_percentile'] + rng.normal(0, 10, len(df))
+
+# calculate the difference between the new perceived percentiles (with randomness) and the actual percentiles
+df['difference_w_randomness'] = df['perceived_percentile_w_randomness'] = df['actual_percentile']
```
Comment (matthew-brett) [inline dunning_kruger.Rmd:112]:
```suggestion
df['difference_w_randomness'] = df['perceived_percentile_w_randomness'] - df['actual_percentile']
```

As you just pointed out yourself!

File: dunning_kruger.Rmd [hunk 3]
```diff
@@ -103,3 +103,21 @@ df
...
+df.head()
+```
+
+```{python}
+
+```
+
+```{python}
+
+```
```
Comment (matthew-brett) [inline dunning_kruger.Rmd:123]:
```suggestion
```

Delete final empty cells.

**Review by matthew-brett — COMMENTED** [review]
Nearly there - thanks.

**Comment by pxr687** [comment]
> Nearly there - thanks.

I've had a look at your review - all looks good to me, thank you for correcting that error!

**Pull request merged**